app/src/pages/playground/schemas.ts

```diff
@@ -104,12 +105,30 @@ const chatMessageSchema = schemaForType<ChatMessage>()(
  */
 export const chatMessagesSchema = z.array(chatMessageSchema);
 
+// https://zod.dev/?id=json-type
```

Comment:
maybe a doc comment with @see so this is a little more clear what this is referring to